src/main/java/com/synopsys/integration/jenkins/JenkinsProxyHelper.java

```diff
@@ -110,4 +128,23 @@ private static boolean shouldIgnoreHost(final String hostToMatch, final List<Pat
         return false;
     }
 
+    @Deprecated
+    public static ProxyInfo getProxyInfoFromJenkins(final String url) {
+        final ProxyConfiguration proxyConfiguration = Optional.ofNullable(Jenkins.getInstanceOrNull())
+                                                    .map(jenkins -> jenkins.proxy)
+                                                    .orElse(null);
```

Comment:
I think the integrations would benefit from these 3 lines (133-135) being put into a separate static method in this class to get a ProxyConfiguration consistently.